examples/interaction/tools/hover_tooltip_advanced.py

```diff
@@ -12,19 +12,22 @@
 colors = np.random.choice(Spectral11, size=N)
 
 p = figure(
-    title="Demonstrates hover tool with advanced and regular tooltip formatting side-by-side",
+    title="Demonstrates hover tool with advanced and regular tooltip formatting and filtering side-by-side",
     tools="pan,wheel_zoom,box_select,crosshair",
 )
 
 p.circle(x, y, radius=radii, fill_color=colors, fill_alpha=0.6, line_color=None)
 
-x_ref = ValueRef(style=dict(background_color="cyan"), field="x")
-y_ref = ValueRef(style=dict(background_color="lime"), field="y")
+x_filter = CustomJS(code="""export default (_, __, {value: x}) => 20 <= x && x <= 80""")
```

Comment:
```suggestion
x_filter = CustomJS(code="""export default (args, tool, {value: x}) => 20 <= x && x <= 80""")
```

Let's put the arguments here even if they are not used.